### PR TITLE
Moved existing tracing context to common class + utils to as…

### PR DESCRIPTION
**Pull request by labbati**

### CODE REVIEW

File: tests/Integration/Common/IntegrationTestCase.php [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+        }
+    }
+
+    /**
+     * Checks that a span expectation is matched in a collection on Spans.
+     *
+     * @param $traces
+     * @param SpanAssertion $exp
+     */
+    public function assertSpan($traces, SpanAssertion $exp)
```
Comment (pawelchcki) [inline tests/Integration/Common/IntegrationTestCase.php]: Have you considered extracting this method to a class that specifically performs the assertions ?

File: tests/Integration/Common/IntegrationTestCase.php [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+        return $transport->getTraces();
+    }
+
+    /**
+     * @param $name string
+     * @param $fn
+     * @return Span[][]
+     */
+    public function inTestScope($name, $fn)
```
[the remark this answers is not in the document]
Comment (pawelchcki) [reply tests/Integration/Common/IntegrationTestCase.php]: Thanks!

**Comment by labbati** [comment]
Made code compatible with 5.4

File: tests/Integration/Common/IntegrationTestCase.php (hunk 1 above)
Comment (labbati) [reply tests/Integration/Common/IntegrationTestCase.php]: This looks like a great idea, let me look into it.

**Comment by pawelchcki** [comment]
:shipit: Looks good!

**Pull request merged**